### ISSUE TITLE
help on optimization restart

### ISSUE THREAD

**Issue by ooodragon94** [huawei-noah/HEBO#48]
sorry, I know this problem belongs to GPy, but I'm new to GP and I can't really solve the problem by myself.
I get following error

Training is completed. Best valid loss:2.801e-01
Warning - optimization restart 2/10 failed
Warning - optimization restart 3/10 failed
Warning - optimization restart 4/10 failed
Warning - optimization restart 5/10 failed
Warning - optimization restart 6/10 failed
Warning - optimization restart 7/10 failed
Warning - optimization restart 8/10 failed
Warning - optimization restart 9/10 failed
Warning - optimization restart 10/10 failed

My search space is following:
        hps = []
        hps.append({'name': 'num_layers', 'type': 'int', 'lb': 3, 'ub': 20})
        hps.append({'name': 'num_nodes', 'type': 'int', 'lb': 8, 'ub': 1024})
        hps.append({'name': 'learning_rate', 'type': 'pow', 'lb': 1e-4, 'ub': 1e-1, 'base': 10})

basically searching for best MLP width, layer and learning rate.
Any possible solution?

thank you in advance

**Comment by ooodragon94** [comment]
additional info:

I found that increasing rand_sample resolves the problem.
However, I'm still not sure why there was no problem on old version with small rand_samples to start with

**Comment by LemurPwned** [comment]
I'm also getting the same warning with the latest version (installed from the repo) as well as the one installed from pypi.
